src/r5py/r5/travel_time_matrix_computer.py

```diff
@@ -104,9 +104,9 @@ def __init__(
         # R5 has a maximum number of destinations for which
         # it returns detailed information, and it’s set
         # at 5000 by default. The value is a static property
-        # of com.conveyal.r5.analyst.cluster.PathResult; can
-        # static properites of Java classes be modified in a
-        # singleton kind of way?)
+        # of com.conveyal.r5.analyst.cluster.PathResult;
+        # static properites of Java classes can be modified
+        # in a singleton kind of way?)
```

Comment:
This entire block is moving into RegionalTask with issue #132 so this will change completely

Reply:
👍🏻